QtCollider/widgets/QcFileDialog.h

```diff
@@ -34,7 +34,8 @@ class QcFileDialog : public QObject
 public:
 
   Q_INVOKABLE QcFileDialog( int fileMode = QFileDialog::ExistingFile,
-                            int acceptMode = QFileDialog::AcceptOpen );
+                            int acceptMode = QFileDialog::AcceptOpen,
+                            const QString& directory = "" );
```

Comment:
i am not sure how to make this default to `QDir::home()`

Reply:
`= QDir::home().path()`